### PR TITLE
Cover attributes based on inputType

### PR DESCRIPTION
**Pull request by andrzejewsky**
ref: https://github.com/saleor/saleor-dashboard/issues/3272

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets and [read good practices](/.changeset/README.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

**Review by krzysztofzuraw — DISMISSED** [review]

**Review by krzysztofzuraw — DISMISSED** [review]

File: src/components/ConditionalFilter/FiltersArea.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const FiltersArea = ({ onConfirm }: FiltersAreaProps) => {
 
   const handleStateChange = async (event: FilterEvent["detail"]) => {
     if (!event) return;
+    console.log("event", event)
```
Comment (poulch) [inline src/components/ConditionalFilter/FiltersArea.tsx:28]:
```suggestion
```

File: src/components/ConditionalFilter/FiltersArea.tsx [hunk 2]
```diff
@@ -43,6 +44,7 @@ export const FiltersArea = ({ onConfirm }: FiltersAreaProps) => {
     }
 
     if (event.type === "rightOperator.onChange") {
+      console.log("rightOperator.onChange", event.value)
```
Comment (poulch) [inline src/components/ConditionalFilter/FiltersArea.tsx:47]:
```suggestion
```

File: src/components/ConditionalFilter/FiltersArea.tsx [hunk 3]
```diff
@@ -51,6 +53,8 @@ export const FiltersArea = ({ onConfirm }: FiltersAreaProps) => {
     }
 
     if (event.type === "rightOperator.onInputValueChange") {
+      console.log("rightOperator.onInputValueChange", event.value)
+
```
Comment (poulch) [inline src/components/ConditionalFilter/FiltersArea.tsx:57]:
```suggestion
```

File: src/components/ConditionalFilter/ValueProvider/TokenArray/fetchingParams.ts [hunk 4]
```diff
@@ -20,6 +20,10 @@ export const emptyFetchingParams: FetchingParams = {
 
 const unique = <T>(array: Iterable<T>) => Array.from(new Set(array));
 
+const includedInParams = (c: UrlToken) =>
+  TokenType.ATTRIBUTE_DROPDOWN === c.type
+  || TokenType.ATTRIBUTE_MULTISELECT === c.type
```
Comment (poulch) [inline src/components/ConditionalFilter/ValueProvider/TokenArray/fetchingParams.ts:25]:
```suggestion
const includedInParams = (c: UrlToken) => [TokenType.ATTRIBUTE_DROPDOWN, TokenType.ATTRIBUTE_MULTISELECT].includes(c.type)
```

File: src/components/ConditionalFilter/ValueProvider/UrlToken.ts [hunk 5]
```diff
@@ -87,7 +107,7 @@ export class UrlToken {
     public value: string | string[],
     public type: TokenTypeValue,
     public conditionKind: string,
-  ) {}
+  ) { }
```
Comment (poulch) [inline src/components/ConditionalFilter/ValueProvider/UrlToken.ts:110]:
```suggestion
  ) {}
```

File: src/components/ConditionalFilter/ValueProvider/UrlToken.ts [hunk 6]
```diff
@@ -100,7 +120,15 @@ export class UrlToken {
...
-    return this.type === TokenType.ATTRIBUTE;
+    const result = Object.entries(TokenType)
+      .find(([_, slug]) => slug === this.type)
+
+    return result && result[0].includes("ATTRIBUTE")
+  }
+
+  public hasDynamicValues() {
+    return TokenType.ATTRIBUTE_DROPDOWN === this.type
+      || TokenType.ATTRIBUTE_MULTISELECT === this.type
```
Comment (poulch) [inline src/components/ConditionalFilter/ValueProvider/UrlToken.ts:131]:
```suggestion
    return [TokenType.ATTRIBUTE_DROPDOWN,  TokenType.ATTRIBUTE_MULTISELECT].includes(this.type)
```

**Review by poulch — APPROVED** [review]

File: src/components/ConditionalFilter/ValueProvider/UrlToken.ts (hunk 6 above)
Comment (andrzejewsky) [reply src/components/ConditionalFilter/ValueProvider/UrlToken.ts:131]: it doesn't match the type

**Review by krzysztofzuraw — APPROVED** [review]

**Review by poulch — APPROVED** [review]

**Pull request merged**